src/libs/Pusher/pusher.ts

```diff
@@ -1,10 +1,36 @@
 import Onyx from 'react-native-onyx';
-import _ from 'underscore';
+import {Channel, ChannelAuthorizerGenerator, Options} from 'pusher-js/with-encryption';
+import isObject from 'lodash/isObject';
 import ONYXKEYS from '../../ONYXKEYS';
-import Pusher from './library';
+import Pusher from './library/types';
```

Comment:
```suggestion
import Pusher from './library';
```

Now we can import correctly here.